index.html

```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
 </div>
 
       </div>
+<div>      
+  <h2 id="google_analytics_tracking">Google Analytics tracking</h2>
+  <p>
+    The eosgi-maven-plugin collect anonym usage statistics. The collected statistics use to obtain budget to further development.
+    Collected data are:
+  </p>
+  <ul>
+    <li>analytics referer: use the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.</li>
+    <li>executed goal: the executed goal.</li>
+    <li>mac address hash: the MAC address hash to connect usage to anonym user. Use SHA-1 hash algorithm.</li>
+    <li>plugin version: the plugin version.</li>
+  </ul>
+  <p>
+    If you want turn off to send usage statistic only add -Deosgi.analytics.skip=true property when build project.
```

Comment:
To turn off the Google Analytics tracking add the "-Deosgi.analytics.skip=true" property to the build comman or use the "<skipAnalytics>true</skipAnalytics>" plugin parameter.